www/js/usePermissionStatus.ts

```diff
@@ -299,11 +299,6 @@ const usePermissionStatus = () => {
         let locExplanation = t('intro.appstatus.overall-loc-description');
         if(window['device'].platform.toLowerCase() == "ios") {
             overallFitnessName = t('intro.appstatus.overall-fitness-name-ios');
-            if(window['device'].version.split(".")[0] < 13) {
-                locExplanation = (t("intro.permissions.locationPermExplanation-ios-lt-13"));
-            } else {
-                locExplanation = (t("intro.permissions.locationPermExplanation-ios-gte-13"));
-            }
```

Comment:
@Abby-Wheelis but if you remove this, then we don't tell people "how to fix" anywhere. If mixing the two is too confusing, maybe we should revisit, and split "why" and "how" across the board. Can we revert for now and revisit in a cleanup?

Reply:
>if you remove this, then we don't tell people "how to fix" anywhere

I think we still do, the issue I identified was that we were telling people "how to fix" but not "why". If you look at the 2nd and 3rd screenshots in the top comment in [Cleanup notes for new onboarding](https://github.com/e-mission/e-mission-docs/issues/1003#top) it shows the instructions for fixing in **both** the permissions modal next to the fix button, and in the explanation popup under location. The changes here only changed what shows in the explanation popup. The text that shows next to the fix button is set around line 156 of this file, and will still show. 

In "how to fix" we show the text from `intro.appstatus.locperms.description.ios-gte-13` (or `lte-12`)
In "why" we show the text from `intro.appstatus.overall-loc-description`

> maybe we should revisit, and split "why" and "how" across the board

Splitting "why" and "how" across the board was part of the changes I made when I first implemented this in React, I just missed this.